fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/socialmedia/SocialMediaViewModel.kt

```diff
@@ -60,6 +61,8 @@ constructor(
   private val mNumberCharsLeft = MutableLiveData<Int>()
   val bottomNavigationTab = MutableLiveData(SocialMediaTab.HOME)
 
+  private val _hasValidToken = MutableLiveData<Boolean>()
```

Comment:
How come you decided to use this name convention here (I mean underscore to start the variable name)

Reply:
It is supposed to be the convention for mutable and private data. Copilot chose the name for me, and it felt normal since I've already come across this kind of naming. But i recon it can be misleading and there should be no issue not following this dark and unknown convention